content/docs/setup/kubernetes/mesh-expansion/index.md

```diff
@@ -5,272 +5,280 @@ weight: 50
+        -o jsonpath='{.data.root-cert\.pem}' |base64 --decode > root-cert.pem
+    $ kubectl -n $SERVICE_NAMESPACE get secret istio.default  \
+        -o jsonpath='{.data.key\.pem}' |base64 --decode > key.pem
+    $ kubectl -n $SERVICE_NAMESPACE get secret istio.default  \
+          -o jsonpath='{.data.cert-chain\.pem}' |base64 --decode > cert-chain.pem
 
-    {{< text bash >}}
-    $ cat kubedns
-    server=/svc.cluster.local/10.150.0.7
-    address=/istio-mixer/10.150.0.8
-    address=/istio-pilot/10.150.0.6
-    address=/istio-citadel/10.150.0.9
-    address=/istio-mixer.istio-system/10.150.0.8
-    address=/istio-pilot.istio-system/10.150.0.6
-    address=/istio-citadel.istio-system/10.150.0.9
     {{< /text >}}
 
 ### Setting up the machines
 
-As an example, you can use the following "all inclusive" script to copy
-and install the setup:
+Next, on each VM you want to add to the mesh:
 
-{{< text bash >}}
-$ export GCP_OPTS="--zone MY_ZONE --project MY_PROJECT"
-$ export SERVICE_NAMESPACE=vm
-{{< /text >}}
+1.  Install the debian package with the Envoy sidecar:
 
-If you are running on a GCE VM, run
-
-{{< text bash >}}
-$ @install/tools/setupMeshEx.sh@ gceMachineSetup VM_NAME
-{{< /text >}}
+    {{< text bash >}}
 
-Otherwise, run
+    $ curl -L https://storage.googleapis.com/istio-release/releases/1.0.0/deb/istio-sidecar.deb > istio-sidecar.deb
+    $ dpkg -i istio-sidecar.deb
 
-{{< text bash >}}
-$ @install/tools/setupMeshEx.sh@ machineSetup VM_NAME
-{{< /text >}}
+    {{< /text >}}
 
-GCE provides better user experience since node agent can always relies on
-GCE metadata instance document to authenticate to Citadel. For everything
-else, e.g., on-prem or raw VM, we have to bootstrap a key/cert as credential,
-which typically has a limited lifetime. And when the cert expires, you have to
-rerun the above command.
+2.  Copy the cluster.env and *.pem files that you created in the previous section to the VM.
 
-Or the equivalent manual steps:
+3.  Add the IP address of the Istio gateway (which we found in the previous section) to `/etc/hosts` or to the DNS server. In our example we'll use `/etc/hosts` as it is the easiest way to get things working. The following is an example of updating an `/etc/hosts` file with the Istio gateway address:
 
------- Manual setup steps begin ------
+    {{< text bash>}}
 
-* Copy the configuration files and Istio Debian files to each machine joining the cluster.
-Save the files as `/etc/dnsmasq.d/kubedns` and `/var/lib/istio/envoy/cluster.env`.
+    $ echo "35.232.112.158 istio-citadel istio-pilot istio-pilot.istio-system" >> /etc/hosts
 
-*   Configure and verify DNS settings. This may require installing `dnsmasq` and either
-adding it to `/etc/resolv.conf` directly or via DHCP scripts. To verify, check that the VM can resolve
-names and connect to pilot, for example:
+    {{< /text >}}
 
-    On the VM/external host:
+4.  Install `root-cert.pem`, `key.pem` and `cert-chain.pem` under `/etc/certs`.
```

Comment:
I would change Install to copy since that is also the command you use:
```suggestion
4.  Copy the `root-cert.pem`, `key.pem` and `cert-chain.pem` files to the `/etc/certs` folder.
```

Reply:
"Install" is a common term used for copying certificate files like .pem files on to a machine, so I think it's ok.